nix/default.nix

```diff
@@ -37,5 +38,5 @@ let
 
 in
 {
-  inherit pkgs plutus sources;
+  inherit pkgs plutus sources ownOverlays;
```

Comment:
why?

Reply:
doesn't make any sense indeed. Dropped it.